docs/setup/manual/route_reflector.md

```diff
@@ -1,12 +1,12 @@
-# Setting Up a Route Reflector
+# Setting Up a route reflector
```

Comment:
We should use consistent casing in all documentation titles. I would prefer Title Case, but that needs to be used everywhere